packages/phone/typings/Phone.d.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Field } from '@availity/form/typings';
+
+interface ExtensionProps extends Field {
+  name: string;
+  id?: string;
+  extColProps?: object;
+}
+export interface PhoneProps extends Field {
```

Comment:
`PhoneProps` should be extending [FieldProps](https://github.com/Availity/availity-react/blob/master/packages/form/typings/Field.d.ts#L6) not field.